<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>431</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>抽奖活动奖项信息</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>331</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>抽奖人数：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>奖项总数：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>已中奖人数：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>奖品名额：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>剩余奖项数：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>剩余奖品名额：</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>开始抽奖</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>331</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnNameList">
        <property name="text">
         <string>名单设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAwardList">
        <property name="text">
         <string>奖项设置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnWinList">
        <property name="text">
         <string>中奖查看</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QListWidget" name="listProjects">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>431</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>抽奖活动基本信息</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>20</y>
       <width>411</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>项目名</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditProjectName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>抽奖主题</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditProjectMainTitle"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>抽奖副标题</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditProjectSubTitle"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>滚动字幕</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditProjectRollTitle"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>背景图片</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelImage">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEditImage">
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>编辑图片</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>groupBox</zorder>
   <zorder>listProjects</zorder>
   <zorder>widget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionSelectAll"/>
    <addaction name="separator"/>
    <addaction name="actionRename"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSearch"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionRegister"/>
    <addaction name="separator"/>
    <addaction name="actionVersion"/>
   </widget>
   <widget class="QMenu" name="menuBingo">
    <property name="title">
     <string>抽奖</string>
    </property>
    <addaction name="actionRestart"/>
    <addaction name="actionContinue"/>
    <addaction name="separator"/>
    <addaction name="actionNameList"/>
    <addaction name="actionAwardList"/>
    <addaction name="actionWinningList"/>
    <addaction name="separator"/>
    <addaction name="actionUILayout"/>
    <addaction name="actionAudio"/>
    <addaction name="actionKeyboard"/>
    <addaction name="actionBingoMode"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuBingo"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="separator"/>
   <addaction name="actionSelectAll"/>
   <addaction name="actionDelete"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>重命名</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>全选</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>粘帖</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>检索</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>版本</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="actionRegister">
   <property name="text">
    <string>注册</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>重新开始抽奖</string>
   </property>
  </action>
  <action name="actionContinue">
   <property name="text">
    <string>继续上次抽奖</string>
   </property>
  </action>
  <action name="actionNameList">
   <property name="text">
    <string>名单设置</string>
   </property>
  </action>
  <action name="actionAwardList">
   <property name="text">
    <string>奖项设置</string>
   </property>
  </action>
  <action name="actionUILayout">
   <property name="text">
    <string>界面设置</string>
   </property>
  </action>
  <action name="actionAudio">
   <property name="text">
    <string>音乐设置</string>
   </property>
  </action>
  <action name="actionKeyboard">
   <property name="text">
    <string>键盘设置</string>
   </property>
  </action>
  <action name="actionBingoMode">
   <property name="text">
    <string>抽奖方式</string>
   </property>
  </action>
  <action name="actionWinningList">
   <property name="text">
    <string>中奖查看</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
